uvtools/dspec.py

```diff
@@ -135,16 +199,127 @@ def delay_filter(data, wgts, bl_len, sdf, standoff=0., horizon=1., min_dly=0.0,
         d_res: best fit high-pass filter components (CLEAN residual) in the frequency domain
         info: dictionary (1D case) or list of dictionaries (2D case) with CLEAN metadata
     '''
+    # get bl delay
+    bl_dly = _get_bl_dly(bl_len, horizon=horizon, standoff=standoff, min_dly=min_dly)
+
+    # run fourier filter
+    return high_pass_fourier_filter(data, wgts, bl_dly, sdf, tol=tol, window=window,
+                                    skip_wgt=skip_wgt, maxiter=maxiter, gain=gain, **win_kwargs)
+
+
+def fringe_filter(data, wgts, max_frq, dt, tol=1e-4, skip_wgt=0.5, maxiter=100,
+                  gain=0.1, window='none', **win_kwargs):
+    """
+    Run a CLEAN deconvolution along the time axis.
+
+    Args:
+        data : 1D or 2D data array. If 2D, shape=(Ntimes, Nfreqs)
+        wgts : 1D or 2D weight array.
+        max_frq : float, maximum frequency [1/sec] to CLEAN
```

Comment:
This is a confusing name (sounds like it's the maximum frequency in the data set). I'd call it max_fr or max_frate or something like that.